apps/password-reset/src/app/pages/password-reset-page/password-reset-page.component.ts

```diff
@@ -1,38 +1,48 @@
 import { Component, Input, OnInit } from '@angular/core';
 import { DomSanitizer, SafeHtml } from '@angular/platform-browser';
 import { StoreService } from '@perun-web-apps/perun/services';
+import { PasswordAction, PasswordLabels } from '@perun-web-apps/perun/models';
```

Comment:
'PasswordLabels' is defined but never used
```suggestion
import { PasswordAction } from '@perun-web-apps/perun/models';
```